client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -466,7 +466,8 @@ export class BpmnEditor extends CachedComponent {
       setColor: !!selectionLength,
       spaceTool: !inputActive,
       undo: commandStack.canUndo(),
-      zoom: true
+      zoom: true,
+      bottomPanel: true
```

Comment:
What is the purpose of this variable? Why do we explicitly need to configure it? I assume that the _bottom panel_ is always available, one way or the other.

Reply:
Agreed. Since the bottom panel is universally accessible, configuring this variable explicitly seems unnecessary.

Reply:
Why would the `bottomPanel` entry point still be in here? We want it to be toggleable from _any_ view.

Reply:
I.e. also from the welcome page.